packages/rpc/src/error.ts

```diff
@@ -0,0 +1,18 @@
+import { RPCErrorCodes } from './typings/types.js';
+
+export class RPCError extends Error {
+	public code: RPCErrorCodes;
+
+	public override message: string = '';
+
+	public override get name() {
+		return `RpcError${RPCErrorCodes[this.code]}`;
```

Comment:
```suggestion
		return `RpcError ${RPCErrorCodes[this.code]}`;
```
wouldn't it be better have `RpcError 42069` than `RpcError42069`